src/dishka/async_container.py

```diff
@@ -110,14 +123,14 @@ def __call__(
     async def get(
             self,
             dependency_type: type[T],
```

Comment:
afaik relates to #171

Reply:
we need overload here, so we can keep usage of `type`. Let it be out of scope for now